### DISCUSSION TITLE
How to change the value of the scroll amount for ctrl-u and ctrl-d?

### DISCUSSION THREAD

**Discussion by abriejenny** [VSCodeVim/Vim#8504] [Q&A] [answered]
Below it says that the default scroll amount is half a screen, I would prefer it to be something like 5 lines, is that possible?
https://github.com/VSCodeVim/Vim/blob/master/ROADMAP.md#scrolling

**Comment by J-Fields** [comment, marked as answer]
See the `vim.scroll` setting

**Comment by abriejenny** [comment]
For other that might land here, it's a top level setting which is just `"vim.scroll": x` where x is amount of lines to scroll.
